modules/alloy/src/test/ts/browser/ui/slider/TwoDSliderTest.ts

```diff
@@ -8,13 +8,12 @@ import { Representing } from 'ephox/alloy/api/behaviour/Representing';
 import * as GuiFactory from 'ephox/alloy/api/component/GuiFactory';
 import * as GuiSetup from 'ephox/alloy/api/testhelpers/GuiSetup';
 import { Slider } from 'ephox/alloy/api/ui/Slider';
-import * as PhantomSkipper from 'ephox/alloy/test/PhantomSkipper';
 
 UnitTest.asynctest('Browser Test: ui.slider.TwoDSliderTest', (success, failure) => {
 
   // Tests requiring 'flex' do not currently work on phantom. Use the remote  to see how it is
```

Comment:
The double space you fixed in `HorizontalSliderTest` is here too

```suggestion
  // Tests requiring 'flex' do not currently work on phantom. Use the remote to see how it is
```

Reply:
This is really outside the scope of this PR... I'll fix it, but I just want to point that out.

Reply:
I only mentioned it as you fixed it in the other file

Reply:
Oh, I can't recall doing that :joy: